librz/include/rz_rop.h

```diff
@@ -0,0 +1,74 @@
+// SPDX-FileCopyrightText: 2024 z3phyr <giridh1337@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef RZ_ROP_H
+#define RZ_ROP_H
+
+#endif // RZ_ROP_H
+
+#include <rz_cmd.h>
+
+typedef struct rz_reg_info_t {
```

Comment:
When you create new include with the API, please document types (structures, enums) using Doxygen syntax as well.

Reply:
Also I recommend a file-level note what this header for